src/command/toggleattributecommand.js

```diff
@@ -43,12 +43,23 @@ export default class ToggleAttributeCommand extends Command {
 		 */
 		this.set( 'value', false );
 
-		this.listenTo( this.editor.document.selection, 'change:attribute', () => {
-			this.value = this.editor.document.selection.hasAttribute( this.attributeKey );
+		// Refresh the value and state of the command on #changesDone to make sure that
+		// the correct state of the command is set initially.
+		// https://github.com/ckeditor/ckeditor5-core/issues/50
```

Comment:
I'd say that we don't have to link to this issue here. This should be the standard way of handling refreshing.